cmd/skopeo/tags.go

```diff
@@ -0,0 +1,96 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/containers/image/docker"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+
+	"io"
+	"strings"
+)
+
+// tagListOutput is the output format of (skopeo tags), primarily so that we can format it with a simple json.MarshalIndent.
+type tagListOutput struct {
+	DockerRepository string   `json:",omitempty"`
+	Tags             []string `json:",omitempty"`
+}
+
+type tagsOptions struct {
+	global *globalOptions
+	image  *imageOptions
+	raw    bool // Output the raw manifest instead of parsing information about the image
+	config bool // Output the raw config blob instead of parsing information about the image
+}
+
+func tagsCmd(global *globalOptions) cli.Command {
+	sharedFlags, sharedOpts := sharedImageFlags()
+	imageFlags, imageOpts := imageFlags(global, sharedOpts, "", "")
+
+	opts := tagsOptions{
+		global: global,
+		image:  imageOpts,
+	}
+
+	return cli.Command{
+		Name:  "docker-list-tags",
+		Usage: "List tags in a Docker v2 API registry and repository specified by the REPOSITORY-NAME",
+		Description: fmt.Sprintf(`
+	Return the list of tags in from the repository in the Docker v2 registry "REPOSITORY-NAME"
+
+	See skopeo(1) section "REPOSITORY NAMES" for the expected format
+	`),
+		ArgsUsage: "REPOSITORY-NAME",
+		Flags:     append(sharedFlags, imageFlags...),
+		Action:    commandAction(opts.run),
+	}
+}
+
+func (opts *tagsOptions) run(args []string, stdout io.Writer) (retErr error) {
+	ctx, cancel := opts.global.commandTimeoutContext()
+	defer cancel()
+
+	if len(args) != 1 {
+		return errors.New("Usage: skopeo docker-list-tags <registry host>:<port>/<repository name>")
```

Comment:
PLEASE just call this list-tags, do not add `docker-`.  If this only works currently with `docker` API, then just document it.  Lets not make the skopeo CLI any more complicated then it already is.